src/ray/object_manager/pull_manager.cc

```diff
@@ -48,14 +48,14 @@ uint64_t PullManager::Pull(const std::vector<rpc::ObjectReference> &object_ref_b
     bundle_it =
         task_argument_bundles_.emplace(next_req_id_++, std::move(deduplicated)).first;
   }
-  RAY_LOG(DEBUG) << "Start pull request " << bundle_it->first
-                 << ". Bundle size: " << bundle_it->second.objects.size();
+  RAY_LOG(INFO) << "Start pull request " << bundle_it->first
```

Comment:
I think you'll want to back out the pull manager changes given the recent refactor